### ISSUE TITLE
Generator

### ISSUE THREAD

**Issue by SpooikypokDev** [dalboris/vpaint#74]
There are some things that are more practical to create by hand than others, especially when working with the smooth tool, so a curve generator could be added, to generate the following impractical-to-create-by-hand-curves:
1) Circles.
2) Onions, the way i call a circle where one side was extruded to create a google-maps-marker-like shape.
3) Lines.
Please note that the tool would only generate these and place them somewhere on the drawing.

**Comment by dalboris** [comment]
Yes, tools to create simple geometric shapes, like is standard in any vector graphics program, will be added. Thanks for opening the issue for this :-) I don't think I would add an "onion" shape by default (this seems too specific), but allowing to edit Bézier curves starting from a circle should make it very easy to create it (~10sec seems reasonable).
